src/server/models/hooksModel.ts

```diff
@@ -0,0 +1,46 @@
+import {firestore} from '../../utils/firestore';
+
+const HOOK_COLLECTION_NAME = 'github-hooks';
```

Comment:
Don't think is the right naming. Payload may be more accurate? https://developer.github.com/webhooks/#payloads